Bembel/src/AnsatzSpace/Glue.hpp

```diff
@@ -485,6 +495,13 @@ struct glue_identificationmaker_<Derived, DifferentialForm::DivConforming> {
           d.coef = coef;
           out.push_back(d);
         }
+      } else {
+        for (int i = 0; i < size_of_edge_dofs; ++i) {
+          GlueRoutines::dofIdentification d;
+          d.dofs.push_back(dofs_e1[i]);
+          d.coef = 0;
```

Comment:
I stumbled over this assignment, as it suggest that the value of d.coef matters. As I understand, this coef is not needed, as the corresponding entry is skipped and the coefficient does not matter. Is my understanding correct?